net/pfSense-pkg-freeradius2/Makefile

```diff
@@ -13,7 +13,8 @@ COMMENT=	pfSense package freeradius2
 LICENSE=	APACHE20
 
 RUN_DEPENDS=	${LOCALBASE}/sbin/radiusd:net/freeradius2 \
-		${LOCALBASE}/bin/bash:shells/bash
+		${LOCALBASE}/bin/bash:shells/bash \
+		${LOCALBASE}/bin/python2.7:lang/python27
```

Comment:
This is not the way to define python dependency. There is a special USES macro for it, as documented at https://www.freebsd.org/doc/en/books/porters-handbook/uses-python.html